go.mod

```diff
@@ -131,7 +131,6 @@ require (
 	github.com/MakeNowJust/heredoc v1.0.0 // indirect
 	github.com/NYTimes/gziphandler v1.1.1 // indirect
 	github.com/antlr4-go/antlr/v4 v4.13.0 // indirect
-	github.com/asaskevich/govalidator v0.0.0-20190424111038-f61b66f89f4a // indirect
```

Comment:
nice... should we add this to unwanted dependencies to make sure we don't accidentally bring it back in?

Reply:
oh, you already did :)